File: category/sort/PGS_42746_가장 큰 수.py

def solution(numbers):
    numbers = list(map(str, numbers))
    numbers.sort(key=lambda x: x*3, reverse=True)
    return str(int(''.join(numbers)))

#다른 풀이
import functools

def comparator(a,b):
    t1= a + b
    t2 = a-b
    return (int(t1)-int(t2)) - (int(t1)-int(t2))

def solution(numbers):
    n = [str(x) for x in numbers]
    n = sorted(n, key = functools.cmp_to_key(comparator), reverse = True)

File: category/sort/PGS_42746_가장 큰 수.js

function solution(numbers) {
    var answer = numbers.map(v=>v+'') // 숫자 배열을 문자열 배열로. 
                        .sort((a,b) => (b+a)*1 - (a+b)*1)
                        .join('');

    return answer[0]==='0'?'0':answer;
}